src/Uno.UI/UI/Xaml/Media/Animation/Timeline.cs

```diff
@@ -414,6 +414,10 @@ public void Dispose()
 			GC.SuppressFinalize(this);
 		}
 
+		void IThemeChangeAware.OnThemeChanged() => OnThemeChangedCore();
+
+		private protected virtual void OnThemeChangedCore() { }
```

Comment:
As you are using an explicit interface implementation, I would not suffix the method name with "Core". Personally, I usually keep that suffix for the private methods that should not be used directly except by the "non-core" version.

Reply:
Any suggestion for this? suffix with `-Impl`?

Reply:
I would just name it `OnThemeChanged` without any suffix.